layers/+spacemacs/spacemacs-layouts/packages.el

```diff
@@ -118,7 +118,8 @@
             persp-reset-windows-on-nil-window-conf nil
             persp-set-last-persp-for-new-frames nil
             persp-save-dir spacemacs-layouts-directory
-            persp-set-ido-hooks t)
+            persp-set-ido-hooks t
+            persp-add-buffer-on-after-change-major-mode 'free)
```

Comment:
These were somewhat, but not entirely in alphabetical order. I didn't want to change them and have lots of unrelated changes in this change to mess w/ blame, but i'd be happy to alphabetize if you think it'd be good.